parts/k8s/cloud-init/artifacts/cse_install.sh

```diff
@@ -284,9 +285,6 @@ installKubeletAndKubectl() {
 pullContainerImage() {
     CLI_TOOL=$1
     DOCKER_IMAGE_URL=$2
-    if [[ -n "${PRIVATE_AZURE_REGISTRY_SERVER:-}" ]]; then
```

Comment:
Why are we getting rid of this?

Reply:
Docker login is only executed when we were extracting hyperkube. Since it is removed for 1.17 and we will be pushing custom kube components to a private azure registry, we will have to move the login function to somewhere else.